parcels/particle.py

```diff
@@ -96,11 +104,16 @@ class JITParticle(Particle):
     """
 
     base_vars = OrderedDict([('lon', np.float32), ('lat', np.float32),
-                             ('xi', np.int32), ('yi', np.int32)])
+                             ('xi', np.int32), ('yi', np.int32),
+                             ('active', np.int32)])
```

Comment:
As above, the C compiler will use an integer under the hood, I believe. Also, a native single-byte boolean might skew memory alignment.